WikiWatershed/tr-55
# Upgrade to Python 3

## Overview

Upgrades the code to Python 3.

Connects #71 
Connects #16 
Connects https://github.com/WikiWatershed/model-my-watershed/issues/3165

## Testing Instructions

* Check out this branch and run `pipenv install --dev`
* Run the tests with `pipenv run nosetests --verbosity=2`
  - [ ] Ensure they pass
